index.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+const Promise = require('bluebird');
+const Trailpack = require('trailpack')
+const Primus = require('primus');
+
+const primusDefaults = {
```

Comment:
This should be externalize to give the user the possibility to configure primus and choose his web socket lib (not force engine.io)